### PR TITLE
DO NOT MERGE - Update k8s build for April R3

### PR DESCRIPTION
**Pull request by nikolaishields**

### CODE REVIEW

File: packages/rke2-kube-proxy-1.21/charts/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: rke2-kube-proxy
 description: Install Kube Proxy.
-version: v1.21.12-rke2r2-build20220428
+version: v1.21.12-rke2r3-build20220505
 appVersion: v1.21.12-rke2r2
```
Comment (briandowns) [inline packages/rke2-kube-proxy-1.21/charts/Chart.yaml:5]: Update this line as well.

Comment (nikolaishields) [reply packages/rke2-kube-proxy-1.21/charts/Chart.yaml:5]: Are your refering to changing line 5 to be:
`appVersion: v1.21.12-rke2r3`

**Review by briandowns — APPROVED** [review]

File: packages/rke2-kube-proxy-1.21/charts/values.yaml [hunk 2]
```diff
@@ -3,8 +3,7 @@
 # image for kubeproxy
 image:
   repository: rancher/hardened-kubernetes
-  tag: v1.21.12-rke2r2-build20220428
-
+  tag: v1.21.12-rke2r3-build20220505
```
Comment (jossemargt-3pillar) [inline packages/rke2-kube-proxy-1.21/charts/values.yaml:6]:
```suggestion
  tag: v1.21.12-rke2r3-build20220511
```

File: packages/rke2-kube-proxy-1.21/charts/Chart.yaml [hunk 3]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 name: rke2-kube-proxy
 description: Install Kube Proxy.
-version: v1.21.12-rke2r2-build20220428
-appVersion: v1.21.12-rke2r2
+version: v1.21.12-rke2r3-build20220505
```
Comment (jossemargt-3pillar) [inline packages/rke2-kube-proxy-1.21/charts/Chart.yaml:4]:
```suggestion
version: v1.21.12-rke2r3-20220511
```

Comment (jossemargt-3pillar) [inline packages/rke2-kube-proxy-1.21/charts/Chart.yaml:4]:
```suggestion
version: v1.21.12-rke2r3-build20220511
```

**Review by brooksn — APPROVED** [review]

**Review by briandowns — APPROVED** [review]

**Pull request closed without merging**